app/models/green_lanes/theme.rb

```diff
@@ -0,0 +1,26 @@
+module GreenLanes
+  class Theme
+    include Her::JsonApi::Model
+    use_api Her::XI_API
+
+    MAX_LENGTH = 50
+
+    attributes :section,
+               :subsection,
+               :theme,
+               :description,
+               :category
+
+    collection_path '/admin/green_lanes/themes'
+
+    def label
+      formatted_label.length > MAX_LENGTH ? "#{formatted_label[0...MAX_LENGTH]}..." : formatted_label
```

Comment:
Rails has a `truncate` method on strings which will do this for you

```ruby
formatted_label.truncate(MAX_LENGTH)
```

Reply:
https://api.rubyonrails.org/classes/String.html#method-i-truncate

Reply:
fixed